texastribune/pixelcite
# Add the ticket

#### What's this PR do?
Adds The Ticket as an option.

#### Why are we doing this? How does it help us?
Helps us with branding for social shares about The Ticket.

#### How should this be manually tested?
Once you get the project running, start it up locally and click on "The Ticket" option. See the navy background with red quotes and white words, as well as The Ticket image at the bottom. Try to save the image to see that it downloads as expected.

#### Are there performance implications? If adding JS can it be done async? Do images/CSS/JS have cache headers set?

#### What are the relevant tickets?
Part of Sprint task: https://basecamp.com/1866004/projects/1175549/todos/178229516

#### Next steps?
* put project up on docker hub
* docker login to the pixelcite box with a machine user
* remove the old docker image from the pixcelcite box
* pull the new image from the hub
* consider adding an image pull to the deployment process

#### Smells?

#### TODOs:
* [x] Approval from art on styles

#### Has the relevant documentation/wiki been updated?
TODO update the README/wiki with new deploy process when it's set up

#### Technical debt note
Same / less once deploy process is updated
